### DISCUSSION TITLE
[FEATURE REQ] Display user defined indexer names on dashboard and releases page

### DISCUSSION THREAD

**Discussion by onedr0p** [autobrr/autobrr#1698] [Ideas]
**Is your feature request related to a problem? Please describe.**

Currently the indexer names on these pages are pulled from the [identifier in the indexer definition](https://github.com/autobrr/autobrr/blob/34d6e0cf609e225fa22261177d4b43c258f03566/internal/indexer/definitions/hd-torrents.yaml#L4)

**Describe the solution you'd like**

I would like for the Dashboard and Releases page to display the indexer names I've set in the indexer settings.

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

**Comment by zze0s** [comment]
Implemented in #1706

**Comment by onedr0p** [comment]
@zze0s it looks like this wasn't done for the releases page. I'm still seeing the identifier being used on that page.

**Reply by zze0s** [reply]
It's done for everything except the Indexer Filter dropdown. 
![image](https://github.com/user-attachments/assets/f9e4c452-529b-4fed-a08e-62c273add835)

**Reply by onedr0p** [reply]
Ah yes, my mistake. Thanks!

**Comment by onedr0p** [comment]
Looks like a PR was opened
https://github.com/autobrr/autobrr/pull/1720
